src/app.module.ts

```diff
@@ -38,12 +37,12 @@ import { OrganizationModule } from './modules/organization/organization.module';
         secure: true,
         auth: {
           user: mail.auth.user,
-          pass: mail.auth.pass
-        }
-      }
+          pass: mail.auth.pass,
+        },
+      },
     }),
     FirebaseModule.forRoot({
-      googleApplicationCredential: './cdentalcaregroupfirebase.json'
+      googleApplicationCredential: './cdentalcaregroupfirebase.json',
```

Comment:
please rebase from develop since this code doesn't exists now

Reply:
I have already changed